### PR TITLE
pages.fr: update outdated pages

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by Nico385412** [comment]
No problem I'll help you when i can

File: pages.fr/common/rsync.md [hunk 1]
```diff
@@ -4,34 +4,30 @@
...
+`rsync --recursive {{chemin/vers/origine}}/ {{chemin/vers/destination}}`
 
-- Transférer le contenu d'un dossier (mais pas le dossier lui-même) d'un hôte distant vers un hôte local :
+- Utiliser le mode archive, résolvant les liens symboliques et ignorant les fichiers déjà transférés sauf si plus récents :
 
-`rsync -r {{hote_distant}}:{{chemin/vers/dossier_distant}}/ {{chemin/vers/dossier_local}}`
+`rsync --archive --update --copy-links {{chemin/vers/origine}} {{chemin/vers/destination}}`
 
-- Transférer un dossier [r]écursivement, dans une [a]rchive pour conserver les attributs, en résolvant les [l]iens symboliques, et ignorant les fichiers déjà transférés sa[u]f si plus récents :
+- Transférer un fichier vers un hôte distant exécutant `rsyncd` et supprimez les fichiers sur la destination qui n'existent pas sur l'hôte distant :
```
Comment (nicokosi) [inline pages.fr/common/rsync.md:27]: I don't understand, it seems incorrect. :thinking:

Comment (nicokosi) [reply pages.fr/common/rsync.md:27]: Suggestion (but not sure if correct):
> - Transférer un fichier vers un hôte distant via `rsyncd` en supprimant les fichiers distants qui n'existent pas dans la source :

**Review by nicokosi — APPROVED** [review]
Looks good to me! 👍

**Comment by nicokosi** [comment]
> > Hello! I've noticed something unusual when checking this PR:
> > 
> > * The page `pages.fr/common/cargo.md` is outdated, based on number of commands.
> > * The page `pages.fr/common/git-branch.md` is outdated, based on number of commands.
> > * The page `pages.fr/common/git-clone.md` is outdated, based on number of commands.
> > * The page `pages.fr/common/git-config.md` is outdated, based on number of commands.
> > * The page `pages.fr/common/grep.md` is outdated, based on number of commands.
> > * The page `pages.fr/common/rsync.md` is outdated, based on number of commands.
> > 
> > Is this intended? If so, just ignore this comment. Otherwise, please double-check the commits.
> 
> @Nico385412 @nicokosi any suggestions for the mentioned pages? Would you be so kind to review this PR as well.

I don't know, but the diff seems OK to me.

**Review by vitorhcl — COMMENTED** [review]
@Nico385412 @nicokosi as a non-French speaker (I'm Brazilian), is the verb tense right in the parts I commented? There is some more verbs that end with r, at least in Portuguese those are verbs in the infinitive. A quote from the style guide:
> - Command and example descriptions on pages in French must use the third person singular present indicative tense (présent de l'indicatif à la troisième personne du singulier). For example, use `Extrait une archive` rather than `Extraire une archive` or `Extrais une archive`.

File: pages.fr/common/rsync.md (hunk 1 above)
Comment (Nico385412) [reply pages.fr/common/rsync.md:27]: I agree with @nicokosi. It sounds better

File: pages.fr/common/cargo.md [hunk 2]
```diff
@@ -16,17 +16,13 @@
 
 `cargo install --list`
 
-- Créer un nouveau binaire ou librairie du projet Rust dans le dossier courant :
+- Créer un nouveau binaire ou librairie du projet Rust dans les dossiers spécifiés (ou dans le dossier courant par défaut) :
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/cargo.md:19]: You're right, it should be something like that

```suggestion
- Crée un nouveau binaire ou librairie du projet Rust dans les dossiers spécifiés (ou dans le dossier courant par défaut) :
```

File: pages.fr/common/cargo.md [hunk 3]
```diff
@@ -16,17 +16,13 @@
...
 
-`cargo init --{{bin|lib}}`
+`cargo init --{{bin|lib}} {{chemin/vers/dossier}}`
 
-- Créer un nouveau binaire ou librairie du projet Rust dans un dossier spécifique :
+- Compiler le projet Rust dans le dossier courant en utilisant le profil release :
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/cargo.md:23]: You're right, it should be something like that

```suggestion
- Compile le projet Rust dans le dossier courant en utilisant le profil release :
```

File: pages.fr/common/rsync.md [hunk 4]
```diff
@@ -4,34 +4,30 @@
 > Peut transférer un ou plusieurs fichiers correspondant à un motif.
 > Plus d'informations : <https://download.samba.org/pub/rsync/rsync.1>.
 
-- Transférer un fichier local vers un serveur distant :
+- Transférer un fichier :
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/rsync.md:7]: You're right, it should be something like that

```suggestion
- Transfère un fichier :
```

File: pages.fr/common/rsync.md [hunk 5]
```diff
@@ -4,34 +4,30 @@
...
 
-`rsync {{chemin/vers/fichier_local}} {{hote_distant}}:{{chemin/vers/dossier_distant}}`
+`rsync {{chemin/vers/origine}} {{chemin/vers/destination}}`
 
-- Transférer un fichier d'un serveur distant vers l'hôte local :
+- Utiliser le mode archive (copier récursivement les répertoires, copier les liens symboliques sans résolution et conserver les autorisations, la propriété et les délais de modification) :
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/rsync.md:11]: You're right, it should be something like that

```suggestion
- Utilise le mode archive (copier récursivement les répertoires, copier les liens symboliques sans résolution et conserver les autorisations, la propriété et les délais de modification) :
```

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**